aliroot-test.sh

```diff
@@ -0,0 +1,7 @@
+package: aliroot-test
+version: v1
+requires:
+  - aliroot
+---
+#!/bin/sh
+aliroot -b BUILD/slc7-x86_64/aliroot/v5-06-28-1/aliroot/test/gun/sim.C
```

Comment:
BUILD is not packaged and you should not be used. Use $ALICE_ROOT or $ALIROOT_ROOT.